src/plonemeeting/portal/core/browser/resources/theme/global.scss

```
@@ -1,421 +1,428 @@
+  content: "!";
+  font-size: 14px;
+  margin-right: 10px;
+  display: inline-block;
+  border-radius: 20px;
+  text-align: center;
+  color: rgba(255, 255, 255, .9);
+  width: 20px;
+  height: 20px;
+  line-height: 20px;
+  text-shadow: 0 -1 pxrgba(0, 0, 0, .5);
+  position: relative;
```

Stylelint (17.9.0)

[error] Duplicate property "position" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)